lib/imagekit/imagekit.rb

```diff
@@ -16,6 +16,15 @@ class ImageKitClient
     attr_reader :file
 
     def initialize(private_key, public_key, url_endpoint, transformation_pos = nil, options = nil)
+      
+      unless(private_key.is_a?(String) && public_key.is_a?(String) && url_endpoint.is_a?(String))
+        raise ArgumentError, "ImageKit Parameters should be valid string"
```

Comment:
We should check the property individually and the error message should help what is missing. Something like this we did in node.js SDK
```
if(!this.options.publicKey) {
        throw new Error(errorMessages.MANDATORY_PUBLIC_KEY_MISSING.message);
    }
    if(!this.options.privateKey) {
        throw new Error(errorMessages.MANDATORY_PRIVATE_KEY_MISSING.message);
    }
    if(!this.options.urlEndpoint) {
        throw new Error(errorMessages.MANDATORY_URL_ENDPOINT_KEY_MISSING.message);
    }
```